SchedulerBot/SchedulerBot.Database.Entities/ScheduledMessageDetails.cs

```diff
@@ -61,5 +57,11 @@ public class ScheduledMessageDetails
 		/// Gets or sets the UTC offset of the recipient time zone.
 		/// </summary>
 		public TimeSpan? TimeZoneOffset { get; set; }
+
+		/// <summary>
+		/// Gets or sets the collection of <see cref="ScheduledMessageDetailsServiceUrl"/> objects
+		/// connecting the current insance to corresponding <see cref="ServiceUrl"/> objects.
```

Comment:
`insance` -> `instance`

Reply:
Good catch, I finally installed ReSpeller after that :D